jwst/ami/utils.py

```diff
@@ -529,54 +409,46 @@ def find_centroid(a):
     if you simulate a psf with pixel_offset = ( (0.2, 0.4), ) then blind
         application  centroid = utils.find_centroid()
 
-    returns the image centroid (0.40036, 0.2000093) pixels in image space. To
+    Returns the image centroid (0.40036, 0.2000093) pixels in image space. To
     use this in lg_model, nrm_core,... you will want to calculate the new image
     center using:
     image_center = utils.centerpoint(s) + np.array((centroid[1], centroid[0])
     and everything holds together sensibly looking at DS9 images of a.
     """
-    ft = matrix_dft.MatrixFourierTransform()
-
-    cv = ft.perform(a, a.shape[0], a.shape[0])
+    cv = matrix_dft(a, a.shape[0], a.shape[0], centering="ADJUSTABLE")
     cvmod, cvpha = np.abs(cv), np.angle(cv)
-
     cvmod = cvmod / cvmod.max()  # normalize to unity peak
-
     htilt, vtilt = findslope(cvpha)
-
     return htilt, vtilt
 
 
 def quadratic_extremum(p):
     """
-    Calculate maximum of the quadratic.
+    Calculate extremum of the quadratic.
 
     Parameters
     ----------
     p : float, float, float
-        Quadratic coefficients
+        Quadratic coefficients p[0]*x*x + p[1]*x + p[2]
 
     Returns
     -------
-    y_max : float
-        Maximum of the quadratic
+    float
+        Extremum of the quadratic
     """
-    y_max = -p[1] / (2.0 * p[0]), -p[1] * p[1] / (4.0 * p[0]) + p[2]
-
-    return y_max
+    return -p[1] / (2.0 * p[0]), -p[1] * p[1] / (4.0 * p[0]) + p[2]
 
 
-def findpeak_1d(yvec, xvec):
+def findpeak_1d(xvec, yvec):
```

Comment:
These inputs were flipped here, but also flipped where this was called, so I flipped both to make this self-consistent and easier to test